src/Tizen.Applications.ComponentManager/Tizen.Applications/ComponentInfo.cs

```diff
@@ -0,0 +1,260 @@
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsIconDisplayed
+        {
+            get
+            {
+                bool iconDisplay = false;
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentInfoIsIconDisplay(_infoHandle, out iconDisplay);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the IsIconDisplay of " + _componentId + ". err = " + err);
+                }
+
+                return iconDisplay;
+            }
+        }
+
+        /// <summary>
+        /// Checks whether the component should be managed by task-manager or not.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsManagedByTaskManager
+        {
+            get
+            {
+                bool managed = false;
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentInfoIsManagedByTaskManager(_infoHandle, out managed);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the IsManagedByTaskManager of " + _componentId + ". err = " + err);
+                }
+
+                return managed;
+            }
+        }
+
+        /// <summary>
+        /// Gets the absolute path of the icon image.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string IconPath
+        {
+            get
+            {
+                string path = string.Empty;
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentInfoGetIcon(_infoHandle, out path);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the IconPath of " + _componentId + ". err = " + err);
+                }
+
+                return path;
+            }
+        }
+
+        /// <summary>
+        /// Gets the label of the component.
+        /// </summary>
+        public string Label
+        {
+            get
+            {
+                string label = string.Empty;
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentInfoGetLabel(_infoHandle, out label);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the Label of " + _componentId + ". err = " + err);
+                }
+
+                return label;
+            }
+        }
+
+        /// <summary>
+        /// Gets the localized label of the component for the given locale.
+        /// </summary>
+        /// <param name="locale">Locale.</param>
+        /// <remarks>The format of locale is language and country code. (avilable value: "[2-letter lowercase language code (ISO 639-1)]-[2-letter lowercase country code (ISO 3166-alpha-2)]")</remarks>
```

Comment:
Spelling, should be `available` instead of `avilable`.

Reply:
Done